lib/srv/forward/sshserver.go

```diff
@@ -268,14 +268,17 @@ func (s *ServerConfig) CheckDefaults() error {
 	if s.TargetServer == nil {
 		return trace.BadParameter("target server is required")
 	}
-	if s.TargetServer.IsOpenSSHNode() {
+	switch s.TargetServer.GetSubKind() {
+	case types.SubKindTeleportNode:
```

Comment:
Should we add a "" case too just to be safe? I'm not sure we enforce that the subkind must be "teleport" for regular node heartbeats.

```suggestion
	case types.SubKindTeleportNode, "":
```